src/TogglePassword/doc/index.rst

```diff
@@ -259,6 +259,35 @@ Then in your form, add your controller as an HTML attribute::
         // ...
     }
 
+Usage without Symfony Forms
+---------------------------
+
+You can also use the TogglePassword with native HTML inputs.
+It requires a little bit more work but it's still possible.
+Here is an example::
+
+    // ...
+
+    <label for="password">Password</label>
+    <input
+        id="password"
+        name="password"
+        type="password"
+        autocomplete="current-password"
+        required
+        placeholder="Password"
+        {{ stimulus_controller('@symfony/ux-toggle-password--toggle-password', {
```

Comment:
```suggestion
        {{ stimulus_controller('symfony/ux-toggle-password/toggle-password', {
```

That should work - the `/` are normalized to `--` with `stimulus_controller`. Also, let's keep off the `@` at the start (it's removed anyway)